recipes/wx/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "wxPython" %}
+{% set version = "4.0.0a2" %}
+{% set sha256 = "3acd5160d69f56b588476d1907eff67b53ad58fb2eba4dedabcdc3490d170338" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: True  # [unix]
```

Comment:
We can probably add the missing packages on Linux via `yum_requirements.txt` I had a recipe here for `wx` but never submitted. I'll see if I can enable Linux in the feedstock.